.github/workflows/nightly-tests.yml

```diff
@@ -189,12 +189,19 @@ jobs:
 
           # prove with no APCs
           MODE="prove-stark" APC=0 ./run.sh
+          echo "Finished proving with no APCs"
           mv metrics.json $RES_DIR/noapc.json
-          mv apcs/apc_candidates.json $RES_DIR/apc_candidates.json
```

Comment:
This was moved to after the 100 apc run because 0 apc does not generate any apcs anymore